xaitk_saliency/impls/gen_image_classifier_blackbox_sal/rise.py

```diff
@@ -57,13 +57,22 @@ def __init__(
             threads=threads
         )
 
+    @property
+    def fill(self) -> Optional[Union[int, Sequence[int]]]:
+        return self._po.fill
+
+    @fill.setter
+    def fill(self, v: Optional[Union[int, Sequence[int]]]) -> None:
+        self._po.fill = v
+
     def _generate(self, ref_image: np.ndarray, blackbox: ClassifyImage) -> np.ndarray:
         return self._po.generate(ref_image, blackbox)
 
     def get_config(self) -> Dict[str, Any]:
         # It turns out that our configuration here is equivalent to that given
         # and retrieved from the RISEGrid implementation that is known set to
         # the internal ``PerturbationOcclusion.perturber``.
-        c = self._po.perturber.get_config()
+        # noinspection PyProtectedMember
+        c = self._po._perturber.get_config()
         c['debiased'] = self._debiased
```

Comment:
That `# noinspection ...` should be in both files from what I am seeing. This doesn't functionally *need* to be there, but suppresses PyCharm from warning about "private" member access when we *know* that the implementation will be setup in a certain way (there could be asserts here instead but they are arguably more expensive and make things a little less readable).

Reply:
We don't need that above the latter of the two because the "private" attribute being accessed is our own attribute. The trigger for the `# no inspection` was due to the `_perturber` access, not the `_po` access.